services/depfu/depfu.service.js

```diff
@@ -1,18 +1,22 @@
 import Joi from 'joi'
-import { BaseJsonService } from '../index.js'
+import { BaseJsonService, redirector } from '../index.js'
 
 const depfuSchema = Joi.object({
   text: Joi.string().required(),
   colorscheme: Joi.string().required(),
 }).required()
 
-export default class Depfu extends BaseJsonService {
+class Depfu extends BaseJsonService {
   static category = 'dependencies'
-  static route = { base: 'depfu', pattern: ':user/:repo' }
+  static route = {
+    base: 'depfu',
+    pattern: ':vcsType(github|gitlab)/:user/:repo',
```

Comment:
With GitLab we can't just use `:user/:repo`, repos can be nested in an aribtrary number of nested subgroups e.g:
https://gitlab.com/megabyte-labs/docker/ci-pipeline/ansible-lint so for projects that work with gitlab we would usually make the route something like `:vcsType(github|gitlab)/:project+` (which allows an arbitrary number of slashes) and then call `https://depfu.com/${vcsType}/shields/${project}` assuming depfu inherits that design descension and exposes it in the same way. Do we have an example of this?